easybuild/easyconfigs/p/poetry/poetry-1.5.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,195 @@
+easyblock = 'PythonBundle'
+
+name = 'poetry'
+version = '1.5.1'
+
+homepage = 'https://python-poetry.org'
+description = """Python packaging and dependency management made easy. Poetry helps you declare, manage and install
+ dependencies of Python projects, ensuring you have the right stack everywhere."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('pkgconf', '1.9.5'),
+    ('Rust', '1.70.0'),  # required for setuptools-rust, which is needed for cryptography
+    ('setuptools-rust', '1.6.0'),
```

Comment:
No longer needed here are they?

Reply:
`Rust` and `setuptools-rust` are no longer needed in `poetry`. Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/18208/commits/a7d8c4005132263f9ab2a0bf45bb6faa80e4f626